CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-## 0.10.15-dev14
+## 0.10.15-dev15
 
 ### Enhancements
 
+* **Adds `chipper` element types to mapping: `Threading`, `Form`, `Field-Name`, `Value`, and `Link`.** Mapped to `NarrativeText` except by `Field-Name` which is mapped to `Title`.
 * **Adds `chipper` element types to mapping: `Headline`, `Subheadline` and `Abstract`.** Mapped respectevely to `Title` (with `category_depth=1`), `Title` (with `category_depth=2`), and `NarrativeText`.
```

Comment:
Maybe combine these two updates since this will be part of the same release?

Reply:
sure, I just changed it